eslint.config.mjs

```diff
@@ -14,7 +14,8 @@ export default defineConfig(
     files: ['src/**/*.ts', 'test/**/*.ts'],
     rules: {
       '@typescript-eslint/ban-ts-comment': 'off',
-      '@typescript-eslint/no-require-imports': 'off'
+      '@typescript-eslint/no-require-imports': 'off',
+      '@typescript-eslint/no-explicit-any': 'off'
```

Comment:
Do we want to do this globally or just allow one line exceptions? Mostly trying to figure out which would give us a better chance of fixing them somewhere down the line